src/components/FormStepSummary/ComponentValueDisplay.js

```diff
@@ -216,7 +216,7 @@ const AddressNLDisplay = ({component, value}) => {
 
   return `${value.postcode} ${value.houseNumber}${value.houseLetter || ''} ${
     value.houseNumberAddition || ''
-  }`;
+  } ${value.city || ''} ${value.streetName || ''}`;
 };
```

Comment:
can we sort out the order to be more natural here?

Modified preview in storybook should look like this:

![image](https://github.com/open-formulieren/open-forms-sdk/assets/5518550/e23b5810-0607-483f-b596-5b37a42226ae)


So that also makes that this should probably be: `$streetName, $houseNumber$houseLetter $houseNumberAddition, $postcode $city`